### PR TITLE
mongodb 3.x supports

### PR DESCRIPTION
**Pull request by newpcraft**
- allow to collect metrics from config, mongos, replica separately
- authentication support
- should define db list in mongodb3_conf.py if need
- require pymongo library due to json parser

### CODE REVIEW

File: collectors/0/mongo3.py [hunk 1]
```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/python
```
Comment (johann8384) [inline collectors/0/mongo3.py]: /usr/bin/env python

File: collectors/0/mongo3.py [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+from collectors.etc import mongodb3_conf
+
+DB_NAMES = []
+CONFIG_CONN = []
+MONGOS_CONN = []
+REPLICA_CONN = []
+
+USER = ''
+PASS = ''
+INTERVAL = 15
```
Comment (johann8384) [inline collectors/0/mongo3.py]: I might have missed it but this should probably come from the config.

**Comment by johann8384** [comment]
Looks pretty good! I made a few small notes, also check your build. It looks like you committed the collector with the wrong permissions for it to be executable.

**Comment by newpcraft** [comment]
It's my mistake, I just updated the code.
Thanks.

**Pull request merged**